File: app/src/main/java/com/icerockdev/babenko/ui/full_screen_image/FullScreenImageInteractor.java
package com.icerockdev.babenko.ui.full_screen_image;

import android.widget.ImageView;

/**
 * Created by Roman Babenko on 14/05/17.
 */

public interface FullScreenImageInteractor {

    void requestImage(ImageView targetIv, String imageUrl,
                      com.squareup.picasso.Callback imageCallback);

}

File: app/src/main/java/com/icerockdev/babenko/ui/full_screen_image/FullScreenImageInteractor.kt
package com.icerockdev.babenko.ui.full_screen_image

import android.widget.ImageView

/**
 * Created by Roman Babenko on 14/05/17.
 */

interface FullScreenImageInteractor {

    fun requestImage(
        targetIv: ImageView, imageUrl: String,
        imageCallback: com.squareup.picasso.Callback
    )

}
